src/main/java/org/apache/commons/dbcp2/BasicDataSource.java

```diff
@@ -1939,6 +1940,8 @@ public synchronized void close() throws SQLException {
             final MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
             try {
                 mbs.unregisterMBean(registeredJmxName);
+            } catch (final InstanceNotFoundException infe) {
+                // Suppress exception
```

Comment:
Hi, 

Thank you for you PR. Why not log a warning here?

Reply:
When using Spring, the MBeans are unregistered before the connection gets closed. So the warning is just noise. The intent of the PR is to suppress the warning.